packages/styled-components/src/utils/error.ts

```diff
@@ -31,10 +31,11 @@ export default function throwStyledComponentsError(
 ) {
   if (process.env.NODE_ENV === 'production') {
     return new Error(
-      `An error occurred. See https://github.com/styled-components/styled-components/blob/main/packages/styled-components/src/utils/errors.md#${code} for more information.${interpolations.length > 0 ? ` Args: ${interpolations.join(', ')}` : ''
+      `An error occurred. See https://github.com/styled-components/styled-components/blob/main/packages/styled-components/src/utils/errors.md#${code} for more information.${
+        interpolations.length > 0 ? ` Args: ${interpolations.join(', ')}` : ''
```

Comment:
Why do we have to check for the array's length?
if we have an empty array and we would call the join method, it would produce an empty string.
if the array will be with at least one item, the join will also perform the job we are looking for.
My suggestion is to call only:
`Args: ${interpolations.join(', ')}`
What do you say?